lib/lambda/sinkMainProcessors.test.ts

```diff
@@ -691,12 +689,10 @@ describe("insertOneMacRecordsFromKafkaIntoMako", () => {
     const expectedRecord = {
       additionalInformation: "info",
       changedDate: null, // due to lastEventTimestamp === 0
```

Comment:
This unit test needs to show that the description/subject of the updated package does not get overwritten by the event happening.  The way this unit test is it just shows what goes from kafka to opensearch, not the end result in opensearch.